### PR TITLE
Pazuzu-137 updated swaggen files according to API changes from registry

### PR DESCRIPTION
**Pull request by boure**
related to #137 and #138 

Updated the swaggen files according to API changes from the registry
+ adapted other files references to compile

To be clear, this PR intends to solve #137 by updating the swaggen files to the latest API of the registry. #138 and the implemetation of the connector itself is out of scope of this PR.

### CODE REVIEW

File: shared/feature.go [hunk 1]
```diff
@@ -24,14 +24,36 @@ type Feature struct {
...
 
 func NewFeature(feature *models.Feature) Feature{
+	var f Feature
+	f.Meta = NewMeta(feature.Meta)
+	f.Snippet = feature.Snippet
+	f.TestSnippet = feature.TestSnippet
+	return f
+}
+
+func NewFeature_str(name string, desc string, auth string, updated time.Time, dependencies []string, snippet string, testSnippet string) Feature{
```
Comment (netme) [inline shared/feature.go]: Not sure that `_str` is a reasonable name here since time is passed using `time.Time` field

Comment (boure) [inline shared/feature.go]: actually time is obsolete
will fix, thanks for pointing it out

File: storageconnector/registryconnector.go [hunk 2]
```diff
@@ -19,7 +21,7 @@ type registryStorage struct {
 	Token string     // OAUTH2 Token
 
 	Features 	*features.Client
-	Files		*files.Client
+	Metas		*feature_metas.Client
```
Comment (netme) [inline storageconnector/registryconnector.go]: minor comment regarding naming. Please use `camelCase` in go

File: storageconnector/registryconnector.go [hunk 3]
```diff
@@ -96,49 +100,71 @@ func (store *registryStorage) SearchMeta(name *regexp.Regexp) ([]shared.FeatureM
 
 // Return a feature metadata from the storage.
 // name:	a value, that must present in feature name
-// TODO : issue #137 -> implement Features Meta on the registry for speed gain
+// TODO untested function !
```
Comment (netme) [inline storageconnector/registryconnector.go]: When leaving a `TODO`, please point to the corresponding issue

**Comment by netme** [comment]
I didn't review the auto-generated code, the rest is almost good

File: storageconnector/registryconnector.go (hunk 2 above)
Comment (boure) [inline storageconnector/registryconnector.go]: not sure if should fix

"feature_metas" actually comes from goswagger applied to "feature-metas"

Comment (netme) [inline storageconnector/registryconnector.go]: Ah, got it, OK

**Comment by netme** [comment]
👍

**Comment by netme** [comment]
Ship it!

**Pull request merged**